examples/balance_bot.cpp

```diff
@@ -207,6 +221,22 @@ void setup()
         }
       });
 
+    /*
+    // make a balance point controller, where p is the balance point. 
+    pid_manager.add_controller(
+      "/balance point", balance_point_params,
+      []()
+      {
+        balance_point_controller.P = balance_point_params.p;
+        balance_point_controller.I = balance_point_params.i;
+        balance_point_controller.D = balance_point_params.d;
+        balance_point_controller.output_ramp =
+            balance_point_params.output_ramp;
+        balance_point_controller.limit = balance_point_params.limit;
+        balancing_lpf.Tf = balance_point_params.lpf_time_constant;
+        balance_point_controller.reset();
+      });
+    */ 
```

Comment:
I'd prefer to not leave commented code in the examples, I want the examples to work as soon as you paste it in with no modifications. I think we should decide to include it or not include it at all. 

On that note, it would be very nice to add very clear comments explaining that this is NOT a PID controller, but simply using P to allow the user to adjust the balance point.

Reply:
I think it's a good thing to include. Hopefully, we'll also support any type of variable in the WEB GUI as well by the time lots of people are using it.

Reply:
ok updated the code and just pushed it. I think this is good to go (sans the API changes you've made)